.editorconfig

```diff
@@ -12,6 +12,12 @@ insert_final_newline = true
 [*.{go,tmpl,html}]
 indent_style = tab
 
+[templates/custom/*.tmpl]
+insert_final_newline = false
```

Comment:
This should be true.

What's the reasoning behind setting this to false? Generally it's really bad form to not end on a newline.

Reply:
Those are the user-editable custom templates that currently are 0-byte files, should I add newlines there?

Reply:
ah!

Sorry

I missed that it was custom!